orbit-2-core/src/main/java/com/babylon/orbit2/ContainerHost.kt

```diff
@@ -25,21 +25,11 @@ interface ContainerHost<STATE : Any, SIDE_EFFECT : Any> {
     /**
      * The orbit [Container] instance.
      *
-     * Use the [Container.create] factory functions to easily obtain a [Container].
+     * Use the [container] factory functions to easily obtain a [Container].
```

Comment:
technically this `[container]` isn't the right one.. it'll point to the `val` below